docs/content/commands/npm-whoami.md

```diff
@@ -6,9 +6,8 @@ description: Display npm username
 
 ### Synopsis
 
-```bash
-npm whoami [--registry <registry>]
-```
+<!-- AUTOGENERATED USAGE DESCRIPTIONS START -->
+<!-- AUTOGENERATED USAGE DESCRIPTIONS END -->
```

Comment:
It looks like this command (and maybe a few others) lost the usage example. I am going to go through and move the usage description for any ones like this to the corresponding .js file.

Reply:
@lukekarrys these are the commands with no usage, the suggested fix is the first comment here.

Reply:
As per this discussion, my [understanding ](https://github.com/npm/cli/issues/4189#issuecomment-1044353640) was that the `*.md` files were out of date. Happy to move that description from the `*.md` to the corresponding `*.js` files. Please lmk what's the next step.

Reply:
No, the `.js` file is up to date.  This is what happens when you run `npm whoami -h`

```sh
npm whoami

Display npm username

Usage:
npm whoami

Options:
[--registry <registry>]
```

The usage output has that `if` statement I commented above to show just `npm whoami` if there is no `usage` attribute on a command.  This is the fix.  The `--registry` flag is just that: a flag, not a parameter. As such it already appears in the `.md` file under [Configuration](https://docs.npmjs.com/cli/v7/commands/npm-whoami#configuration)

All we need to do is add that if statement to this script so that if there is no usage, the `npm [command name]` line is added as the usage example.

Reply:
Got it! Will send an update asap. Thank you for the clarification 👍

Reply:
Thanks @manekinekko! I misread and apologize for the confusion I caused.